openapi/components/schemas/Coupon/CouponRestriction.yaml

```
@@ -13,6 +13,9 @@ discriminator:
     restrict-to-products: ./CouponRestrictions/restrict-to-products.yaml
     restrict-to-subscriptions: ./CouponRestrictions/restrict-to-subscriptions.yaml
     total-redemptions: ./CouponRestrictions/total-redemptions.yaml
+    restrict-to-invoice-items-with-plans: ./CouponRestrictions/restrict-to-invoice-items-with-plans.yaml
+    minimum-item-quantity: ./CouponRestrictions/minimum-item-quantity.yaml
+    minimum-total-item-quantity: ./CouponRestrictions/minimum-total-item-quantity.yaml
```

```suggestion
    minimum-invoice-quantity: ./CouponRestrictions/minimum-total-item-quantity.yaml
```
I propose only adding this one. It applies across items.

For items, I would propose we add an additional property, with default value `1` to:

- restrict-to-products
- restrict-to-plans

The property can be `minimumQuantity` with a default value of 1 (which matches the current behavior).